contrib/vcloud/vcloudbenchmarkbase.py

```diff
@@ -0,0 +1,95 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.benchexec
+import benchexec.model
+import benchexec.tooladapter
+import benchexec.tools
+import benchexec.util
+
+
+class VcloudBenchmarkBase(benchexec.benchexec.BenchExec):
+    """
+    An extension of BenchExec that supports executing the benchmarks in the
+    VerifierCloud (internal project at https://gitlab.com/sosy-lab/software/verifiercloud/).
+    """
+
+    def create_argument_parser(self):
+        parser = super(VcloudBenchmarkBase, self).create_argument_parser()
+        vcloud_args = parser.add_argument_group("Options for using VerifierCloud")
+        vcloud_args.add_argument(
+            "--vcloud",
```

Comment:
Wasn't this parameter removed?

Reply:
that's what I remember too. Anyways, removed it.